### DISCUSSION TITLE
My account has been flagged and I don't know why

### DISCUSSION THREAD

**Discussion by shengxj1** [community/community#123084] [Other Feature Feedback, Questions, & Ideas]
I have already applied for a ticket in the support service, with the ticket number 2737598

https://support.github.com/ticket/personal/0/2737598

Does your claim involve content on GitHub or npm.js?

GitHub

What is the username and repository or package name that was impacted?

shengxinjing

Why are you requesting reinstatement? If you are requesting support for your account not related to moderation limits made on your account by GitHub please raise a support ticket instead.

I can login, but my profile and contributions aren't visible to others

Have you previously contacted GitHub about this claim?

Yes

Please provide your previous ticket number(s) if you can. (Don’t worry if you don’t know the number.)

Would you like to provide any additional information or context that would be helpful for our review of your reinstatement request?

yes
I'm not quite sure why my github is flagged
You are marked as spam, and therefore cannot authorize a third party application.

I have reviewed and understand the GitHub Acceptable Use Policies and Community Guidelines.

![image](https://github.com/community/community/assets/19541668/696b8244-cf84-4272-98cb-be7108cc5c6a)

**Comment by shengxinjing** [comment]
就是我的账号 求助求助

**Comment by shengxj1** [comment]
my account: https://github.com/shengxinjing

**Reply by sajadh76** [reply]
Hi!

A flagged account and related queries that are account-specific should go to [support team](https://support.github.com/). They're the ones who are best able to assist in these situations. The community team and its members are unable to assist!

**Reply by shengxj1** [reply]
Thanks for the reminder. I have appealed to the support team for two weeks and no one has responded to me.

**Reply by PleshakovMihail** [reply]
I have the same problem. 3 tickets, 1.5 months and not a single answer.

**Comment by JimLiu** [comment]
Hopefully someone can help with this.

**Comment by o7q** [comment]
Same here.

**Comment by ghostinhershell** [comment]
Hi @woniuppp,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by shengxj1** [reply]
thanks
I have been creating the ticket for two months now, but there are still no GitHub staff to handle it. I am not sure how to proceed with this process, so I am inquiring here

Can I do anything else besides waiting? Restore my account to normal
